### PR TITLE
Task 108859: Fix cypress tests which  are failing with tags

### PR DESCRIPTION
**Pull request by Lisa-V-Sau**
Fixed legal requirements tests which were failing:

1. Using page objects to locate project
2. Tests are directed to Dev only since the change-link tag is missing on staging

Updated:

- README on tags

### CODE REVIEW

File: ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/e2e/96066_TL_legal-requirements/105392_legal-requirements.cy.js [hunk 1]
```diff
@@ -1,13 +1,14 @@
...
 
-const url = Cypress.env('url') + '/task-list/2054'
-
-describe('Legal Requirements', { tags: ['@dev', '@stage']}, () => {
+describe('Legal Requirements', { tags: '@dev'}, () => {
     beforeEach(() => {
-        cy.visit(url)
-        cy.get('[data-cy="select-tasklist-links-legalrequirements"]').click()
+        legalRequirements.selectProject().then(() =>{
+            cy.url()
```
Comment (jordan-beard) [inline ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/e2e/96066_TL_legal-requirements/105392_legal-requirements.cy.js:9]: What is the cy.url() used for?

File: ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/pages/legalRequirement.js [hunk 2]
```diff
@@ -0,0 +1,14 @@
+class legalRequirements {
+    selectProject() {
```
Comment (jordan-beard) [inline ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/pages/legalRequirement.js:2]: Is this function selecting a project?
It looks like its doing something else/

File: ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/e2e/96066_TL_legal-requirements/105392_legal-requirements.cy.js (hunk 1 above)
Comment (Lisa-V-Sau) [reply ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/e2e/96066_TL_legal-requirements/105392_legal-requirements.cy.js:9]: Its a function that returns the method of locating conversion, project(0).

File: ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/pages/legalRequirement.js (hunk 2 above)
Comment (Lisa-V-Sau) [reply ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/pages/legalRequirement.js:2]: It's a function, something I missed?

File: ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/e2e/96066_TL_legal-requirements/105392_legal-requirements.cy.js (hunk 1 above)
Comment (Lisa-V-Sau) [reply ApplyToBecomeInternal/ApplyToBecomeCypressTests/cypress/e2e/96066_TL_legal-requirements/105392_legal-requirements.cy.js:9]: Good catch! Updating it now.

**Review by jordan-beard — APPROVED** [review]

**Pull request merged**